### DISCUSSION TITLE
New Exception on Running Workflow

### DISCUSSION THREAD

**Discussion by Mike-E-angelo** [community/community#25180] [Actions]
<p>Hello Community,</p>
<p>We have a workflow file that <a href="https://github.com/ExtendedXmlSerializer/home/actions/runs/62994799/workflow" rel="nofollow noopener">can be found here</a>.  It hasn’t been run in about two months now, and the only edit that was provided was for <a href="https://github.community/t/filter-on-release-edited-but-not-while-published/17563/10">outputting debugging values for this issue here</a>.</p>
<p>Upon running it this morning for a new release, I got a new error and I have no idea what it is complaining about. :slight_smile:</p>
<p>Here is the run:</p>
<p><a href="https://github.com/ExtendedXmlSerializer/home/runs/533424140?check_suite_focus=true" rel="nofollow noopener">https://github.com/ExtendedXmlSerializer/home/runs/533424140?check_suite_focus=true</a></p>
<p>Here is the last successful run of this job:</p>
<p><a href="https://github.com/ExtendedXmlSerializer/home/runs/431544119?check_suite_focus=true" rel="nofollow noopener">https://github.com/ExtendedXmlSerializer/home/runs/431544119?check_suite_focus=true</a></p>
<p>As you can see nothing has changed in the file for this job so I am not sure exactly what is happening here and would appreciate some context if at all possible.</p>
<p>Thank you for any assistance you can provide!</p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/mike-e-wins">@mike-e-wins</a> ,</p>
<p>The problem occurs when installing the module <strong>Poshstache. </strong> According to <a href="https://github.com/baldator/Poshstache/issues/10" rel="nofollow noopener">this issue</a>, it looks likes the problem occurs on <strong>PowerShell 7</strong>.</p>
<p>I also tested and tried on  <strong>PowerShell 7</strong> and <strong>PowerShell 5.1</strong> , the module <strong>Poshstache </strong> can be installed successfully on <strong>PowerShell 5.1</strong> but failed on  <strong>PowerShell 7</strong> , same as  <strong>chwarr</strong> mentioned in that issue. And it seems also can work on some other versions lower than <strong>PowerShell 7</strong>.</p>
<p>I noticed that the version of PowerShell on teh GitHub-hosted Ubuntu runner has been updated from  <strong>PowerShell 6.2</strong> to <strong>PowerShell 7</strong> recently (see <a href="https://github.com/actions/virtual-environments/commit/99478a7e217f34c4b88736bbc02625d4a7b94cab#diff-43bac3def16599ef5b79ea8b7d9e0a9aL181" rel="nofollow noopener">here</a>). Maybe this is the reason why your action can’t work now but could work last month.</p>
<p>I have reported this problem to the appropriate engineering team, the team will evaluate and investigate this issue. If they have any progress, we will notify you in time, and sometimes the appropriate engineers may directly reply you here.</p>
<p>You also can follow the issue I mentioned above, and add your comments here.</p>

**Comment by Mike-E-angelo** [comment]
<p>Ahhhh that would do it, <a class="mention" href="/brightran">@brightran</a> !  Thank you for taking the time to investigate and to let me know.  Obviously, it would seem that an upgrade to a new version of a dependency would be an opt-in choice/activity so to avoid scenarios exactly like this.  Now my build is broken without much recourse.  Or an obvious one, anyways. :stuck_out_tongue:</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/mike-e-wins">@mike-e-wins</a> ,</p>
<p>The issue ticket I reported to the appropriate engineering team is <a href="https://github.com/actions/runner/issues/387" rel="nofollow noopener">here</a>. It is on the public repository, so you also can directly follow this issue ticket.</p>

**Comment by Mike-E-angelo** [comment]
<p>Thank you for all your assistance and efforts here <a class="mention" href="/brightran">@brightran</a>.  It is much appreciated. :+1:</p>
